build.gradle

```diff
@@ -18,16 +18,19 @@ buildscript {
     repositories {
         maven { url "https://maven-central.storage.googleapis.com"}
         maven { url "https://plugins.gradle.org/m2/" }
+        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+        maven { url 'http://dl.bintray.com/jetbrains/intellij-plugin-service'  }
     }
     dependencies {
         classpath "net.ltgt.gradle:gradle-errorprone-plugin:0.0.13"
         classpath "net.ltgt.gradle:gradle-apt-plugin:0.13"
+        classpath "org.jetbrains.intellij.plugins:gradle-intellij-plugin:0.4.0-SNAPSHOT"
```

Comment:
any idea from their github page when this will make the release? It would be nice to not include a snapshot version

Reply:
Nothing particular on their page, no due date for that 0.4.0. I agree using snapshot is a dangerous thing, we might just use for community build although I'm not sure this is possible conditionally with Gradle.

Reply:
yeah. with all the headache we've been having the gradle-intellij-plugin I'm worried about making things worse with a snapshot version. I'm tempted to not merge in this fix until its been released (and we'll just be aware of how to do this locally if we need to build with CE). wdyt?

Reply:
also, it would be nice to have a CI build against CE. I'll make an issue for that.

Reply:
Yeah, I think this might be a good idea. We won't be able to add CE into our master target list though like we do with previous and EAP version.

Reply:
why not?

Reply:
Well, we can add that for sure, it's just it will continue to fail for some time until we switch to fresh release, I meant couldn't add target that succeeds for now.

Reply:
oh right. Yeah, I meant after we figure out this whole mess, so that we can catch future messes :)